src/k8s-engine/wsl.ts

```diff
@@ -853,18 +737,24 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     const credsPath = getServerCredentialsPath();
 
     try {
+      const vtunnelPeerServer = '127.0.0.1:3030';
       const hostIPAddr = wslHostIPv4Address();
       const stateInfo: ServerState = JSON.parse(await fs.promises.readFile(credsPath, { encoding: 'utf-8' }));
       const escapedPassword = stateInfo.password.replace(/\\/g, '\\\\')
         .replace(/'/g, "\\'");
       // leading `$` is needed to escape single-quotes, as : $'abc\'xyz'
       const leadingDollarSign = stateInfo.password.includes("'") ? '$' : '';
-      const fileContents = `CREDFWD_AUTH=${ leadingDollarSign }'${ stateInfo.user }:${ escapedPassword }'
-CREDFWD_URL='http://${ hostIPAddr }:${ stateInfo.port }'
-`;
+      const fileContents = `CREDFWD_AUTH=${ leadingDollarSign }'${ stateInfo.user }:${ escapedPassword }'CREDFWD_URL='${ vtunnelPeerServer }'`;
```

Comment:
What happened to the new line? This is a shell script so that's kind of required…

The `http://` is also missing now.